### PR TITLE
[integration] Testnet e2e integration test

### PR DESCRIPTION
**Pull request by jakekidd**
## What is?

Made this test to improve visibility, accountability, and ease-of-use for conducting e2e. Can be run with a local router/sequencer pair or using the live hosted router/sequencer instances.

Additionally, tidied up logs a bit across the board, and added some configuration options to make running locally a lot more feasible.

Partly (?) addresses #929.

## How do?
Check out the README, it's like, actually informative

### CODE REVIEW

File: packages/agents/router/src/router.ts [hunk 1]
```diff
@@ -97,10 +98,23 @@ export const makeRouter = async () => {
...
     await bindCache();
 
-    logger.info("Router ready!");
+    logger.info("Bindings initialized.");
+
+    console.log(
+      "Router boot complete!",
+      `
+
+        _|_|_|     _|_|     _|      _|   _|      _|   _|_|_|_|   _|      _|   _|_|_|_|_|
```
Comment (rhlsthrm) [inline packages/agents/router/src/router.ts:110]: <3

**Review by rhlsthrm — APPROVED** [review]
Definitely should try to pull from deployments but other than that, looks great!

File: packages/agents/router/src/router.ts [hunk 2]
```diff
@@ -97,10 +98,23 @@ export const makeRouter = async () => {
...
     await bindServer();
     await bindMetrics();
-    await bindSubgraph(context.config.subgraphPollInterval!);
+    await bindSubgraph();
     await bindCache();
 
-    logger.info("Router ready!");
+    logger.info("Bindings initialized.");
+
+    console.log(
```
Comment (rhlsthrm) [inline packages/agents/router/src/router.ts:106]: Why is this not logger.info?

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/agents/router/src/router.ts (hunk 2 above)
Comment (jakekidd) [reply packages/agents/router/src/router.ts:106]: I can make the message logger.info, but the connext branding thing has to be console.log or it literally will just show up incoherently in the logs haha

**Pull request merged**